src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -137,6 +136,14 @@ public void reportTrack(String eventName) {
         mMessageThreadHandler.sendMessage(m);
     }
 
+    @Override
+    public void sendErrorMessage(JSONObject errorMessage) {
```

Comment:
This argument probably shouldn't be JSON - the interface could be any listener, so we can use a real object with good static types.